tensorflow/lite/delegates/webnn/webnn_delegate.cc

```diff
@@ -2832,6 +3014,156 @@ class Subgraph {
     return kTfLiteOk;
   }
 
+  static TfLiteStatus VisitStridedSliceNode(
+      const emscripten::val& builder, TfLiteContext* logging_context,
+      int node_index, TfLiteNode* node, const TfLiteTensor* tensors,
+      const TfLiteStridedSliceParams* params,
+      std::unordered_map<int, emscripten::val>& webnn_operands,
+      const bool detect_supported_op) {
+    // Only support strided slice with no ellipsis mask, no new axis mask, and
+    // no shrink_axis-mask.
+    if (params->ellipsis_mask != 0 || params->new_axis_mask != 0 ||
+        params->shrink_axis_mask != 0) {
+      return kTfLiteError;
+    }
+
+    const int stride_tensor_id = node->inputs->data[3];
+    const TfLiteTensor& stride_tensor = tensors[stride_tensor_id];
+
+    TF_LITE_ENSURE_STATUS(CheckShapeTensorShape(logging_context, stride_tensor,
+                                                stride_tensor_id, node_index));
+    TF_LITE_ENSURE_STATUS(CheckTensorStaticAllocation(
+        logging_context, stride_tensor, stride_tensor_id, node_index));
+    TF_LITE_ENSURE_STATUS(CheckTensorInt32Type(
+        logging_context, stride_tensor, stride_tensor_id, node_index));
+
+    const int num_dims = stride_tensor.dims->data[0];
+
+    // Only support strides = 1.
+    auto stride_data = GetTensorData<int32_t>(&stride_tensor);
+    for (size_t i = 0; i < num_dims; i++) {
+      if (stride_data[i] != 1) {
+        TF_LITE_MAYBE_KERNEL_LOG(logging_context,
+                                 "stride at dimension %d, %d, must be 1"
+                                 "in STRIDED_SLICE node #%d",
+                                 i, stride_data[i], node_index);
+        return kTfLiteError;
+      }
+    }
+
+    const int input_tensor_id = node->inputs->data[0];
+    const int begin_tensor_id = node->inputs->data[1];
+    const int end_tensor_id = node->inputs->data[2];
+    const int output_tensor_id = node->outputs->data[0];
+    const TfLiteTensor& input_tensor = tensors[input_tensor_id];
+    const TfLiteTensor& begin_tensor = tensors[begin_tensor_id];
+    const TfLiteTensor& end_tensor = tensors[end_tensor_id];
+    const TfLiteTensor& output_tensor = tensors[output_tensor_id];
+
+    TF_LITE_ENSURE_STATUS(CheckShapeTensorShape(logging_context, begin_tensor,
+                                                begin_tensor_id, node_index));
+    TF_LITE_ENSURE_STATUS(CheckTensorStaticAllocation(
+        logging_context, begin_tensor, begin_tensor_id, node_index));
+    // TODO(b/246969669): TFLite only supports int32 begin ends and strides,
```

Comment:
What does `b/246969669` mean?

Reply:
Umm, seems a TFLite internal bug jira link, I just referred from https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/delegates/xnnpack/xnnpack_delegate.cc#L5404.

Reply:
Consider posting a pointer to xnnpack_delegate source?

Reply:
Sure. Done.

Reply:
The pointer looks good. Please remove `b/246969669` within the TODO.

Reply:
Done.:)